src/psyclone/tests/psyir/backend/c_test.py

```diff
@@ -310,10 +313,9 @@ def test_cw_unaryoperator():
         assert cwriter(unary_operation) in expected
 
     # Test that an unsupported operator raises an error
-    # pylint: disable=abstract-method, too-few-public-methods
-    class Unsupported():
-        '''Dummy class'''
-    # pylint: enable=abstract-method, too-few-public-methods
+    # pylint: disable=too-few-public-methods,missing-docstring
+    class Unsupported(object):
+        pass
     unary_operation._operator = Unsupported
```

Comment:
Do you need to re-enable 'missing-docstring' and 'too-few-public-methods' here? In fact, I think the latter one could be put in the `tests/.pylintrc` file.

Reply:
I did indeed put them in the wrong location. I moved them into the class, which then automatically re-enables them once the class-scope is left.